src/Microsoft.NET.Sdk.Aspire/WorkloadManifest.in.targets

```diff
@@ -5,4 +5,14 @@
     Condition=" ('$(IsAspireHost)' == 'true') and ('$(SkipAspireWorkloadManifest)' != 'true') "
   />
 
+  <Import
+    Project="Sdk.targets" Sdk="Aspire.Hosting.Orchestration"
```

Comment:
nit: We should maybe also import the `Sdk.props` for all the sdk imports here, even though they are empty projects right now.

Reply:
By convention WorkloadManifest.targets files only import targets files as we can't load .props files at the appropriate time.